### PR TITLE
DB task | Sanjar

### PR DESCRIPTION
**Pull request by abstract000**

### CODE REVIEW

File: Abdufattokhov-Sanjar/01-db-task.sql [hunk 1]
```diff
@@ -0,0 +1,113 @@
+CREATE TABLE courses (
```
Comment (sabirjahn) [inline Abdufattokhov-Sanjar/01-db-task.sql]: create table if not exists bilan yozing

File: Abdufattokhov-Sanjar/01-db-task.sql [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+-- 3.1.1: Display all students with their first name, last name, and date of birth
+SELECT first_name, last_name, birthdate FROM students;
+
+-- 3.1.2: Find all students enrolled in the "Tarix" course
+SELECT s.first_name, s.last_name 
+FROM students s
+JOIN enrollments e ON s.student_id = e.student_id
+JOIN courses c ON e.course_id = c.course_id
+WHERE c.course_name = 'Tarix';
```
Comment (sabirjahn) [inline Abdufattokhov-Sanjar/01-db-task.sql]: Mathematics bo'lishi kerak emasmidi?

File: Abdufattokhov-Sanjar/01-db-task.sql [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+ORDER BY last_name;
+
+-- 3.4.2: Find all students enrolled after 2015 and enrolled in the "Tarix" course
+SELECT s.first_name, s.last_name 
+FROM students s
+JOIN enrollments e ON s.student_id = e.student_id
+JOIN courses c ON e.course_id = c.course_id
+WHERE s.enrollment_year > 2015 AND c.course_name = 'Tarix';
+
+-- 3.5: Find the courses taken by the student with the lowest average grade
```
Comment (sabirjahn) [inline Abdufattokhov-Sanjar/01-db-task.sql]: 3.5.2 bor yaxshi lekin 3.5.1 yo'q
- Find students enrolled in more courses than the average number of courses per student.

File: Abdufattokhov-Sanjar/01-db-task.sql [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+SELECT first_name, last_name 
+FROM students
+ORDER BY last_name;
+
+-- 3.4.2: Find all students enrolled after 2015 and enrolled in the "Tarix" course
+SELECT s.first_name, s.last_name 
+FROM students s
+JOIN enrollments e ON s.student_id = e.student_id
+JOIN courses c ON e.course_id = c.course_id
+WHERE s.enrollment_year > 2015 AND c.course_name = 'Tarix';
```
Comment (Nasa-133) [inline Abdufattokhov-Sanjar/01-db-task.sql:81]: enrollment_year ustuni students jadvalida mavjud emas

File: Abdufattokhov-Sanjar/01-db-task.sql [hunk 5]
```diff
@@ -13,74 +22,63 @@ CREATE TABLE enrollments (
...
 
--- 3.4.2: Find all students enrolled after 2015 and enrolled in the "Tarix" course
 SELECT s.first_name, s.last_name 
 FROM students s
 JOIN enrollments e ON s.student_id = e.student_id
 JOIN courses c ON e.course_id = c.course_id
-WHERE s.enrollment_year > 2015 AND c.course_name = 'Tarix';
+WHERE s.enrollment_year > 2015 AND c.course_name = 'Mathematics';
 
--- 3.5: Find the courses taken by the student with the lowest average grade
```
Comment (sabirjahn) [inline Abdufattokhov-Sanjar/01-db-task.sql]: 3.5. Working with Subqueries
Find students enrolled in more courses than the average number of courses per student. - bu bajarilmagan
List the names of courses enrolled by students with the lowest average grade. - bajarilgan

**Pull request closed without merging**